Player/GrenadeLauncher.gd

```diff
@@ -72,78 +71,73 @@ func update_aim() -> void:
 	var motion_down := to_target.y - peak_height
 	var time_going_down := sqrt(-2.0 * motion_down / gravity)
 	
-	var time_to_land := time_going_up + time_going_down
+	_time_to_land = time_going_up + time_going_down
```

Comment:
We need the time to land for our calculations in the draw function so Iǘe changed it to a property

Reply:
I was hesitant to merge the two functions into one and remove the need for this extra property, but perhaps it's a bit too long for people to read